app/assets/javascripts/Results/context_menu.js.erb

```diff
@@ -16,35 +16,39 @@ var annotation_context_menu = {
   setup: function(annot_path, result_id, assignment_id, file_dl_path) {
     var menu_items = {
       new_annotation: {
-        title: 'New Annotation',
+        title: '<%= I18n.t('marker.annotation.context_menu.new') %>',
```

Comment:
It also would have been better to submit these translations in a separate branch.